.kitchen.yml

```diff
@@ -1,21 +1,65 @@
+<%
+rpm_based_platforms = %w(centos-5 centos-6 centos-7 fedora-25)
+deb_based_platforms = %w(ubuntu-12.04 ubuntu-14.04 ubuntu-16.04 debian-7 debian-8)
+opensuse_platforms = %w(opensuse-13.2 opensuse-leap)
+%>
+---
 driver:
   name: vagrant
+  customize:
+    cableconnected1: 'on'
 
 provisioner:
   name: chef_zero
 
+verifier:
+  name: inspec
+
 platforms:
-  - name: centos-6.8
-  - name: centos-7.2
-  - name: debian-7.11
-  - name: debian-8.6
+  - name: debian-7
+    driver:
+      box: bento/debian-7.11
+
+  - name: debian-8
+    driver:
+      box: bento/debian-8.5
+
+  - name: centos-5
+    driver:
+      box: bento/centos-5.11
+
+  - name: centos-6
+    driver:
+      box: bento/centos-6.8
+
+  - name: centos-7
+    driver:
+      box: bento/centos-7.2
+
   - name: fedora-25
+    driver:
+      box: bento/fedora-25
     run_list: yum::dnf_yum_compat
```

Comment:
Yep this isn't necessary with the new chef clients that include support for DNF out of the box